src/app/app.component.ts

```diff
@@ -8,15 +10,14 @@ import { BallDontLieService } from './core/services/ball-dont-lie.service';
 })
 export class AppComponent implements OnInit {
   title = 'random-stats';
-  allTeams: any;
+  allTeams$: Observable<ITeam[]> = of([]);
 
   constructor(
     private ballDontLieService: BallDontLieService
   ) {}
 
 
   ngOnInit(): void {
-    this.allTeams = this.ballDontLieService.getAllNBATeams();
-    console.log("allTeams", this.allTeams)
+    this.allTeams$ = this.ballDontLieService.getAllNBATeams();
```

Comment:
Let's move this into the constructor and remove the ngOnInit and its 'implements'.

That way we don't have to initialize the property and it can just be `allTeams$: Observable<ITeam[]>;` for line 13.